src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.html

```diff
@@ -38,14 +38,14 @@ <h3>
                     <hr />
                     <ul class="list-unstyled">
                         <li class="list-group-item border-1">
-                            <span>
-                                <jhi-exam-checklist-check [checkAttribute]="exam.exerciseGroups?.length > 0"></jhi-exam-checklist-check>
+                            <span *ngIf="exam.exerciseGroups">
+                                <jhi-exam-checklist-check [checkAttribute]="exam.exerciseGroups.length > 0"></jhi-exam-checklist-check>
```

Comment:
This changes the behavior of the component. jhi-exam-checklist-check shows different colors normally but it is full removed here